fix: PLT-809: Fix json_min memory usage

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/15932088426)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/483
<!-- Follow Merge -->

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15932088426

triklozoid commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/15942592055)
> Successfully merged:  create mode 100644 label_studio/tasks/migrations/0055_task_proj_octlen_idx_async.py

triklozoid committed:
Merge branch 'develop' into 'fb-PLT-809'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15942592055

triklozoid commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/16029637180)
> Successfully merged:  create mode 100644 web/libs/ui/src/assets/icons/upright-pin.svg

triklozoid committed:
Merge branch 'develop' into 'fb-PLT-809'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16029637180

triklozoid commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/16046181323)
> Successfully merged:  create mode 100644 .github/workflows/yarn-docs.yml

triklozoid committed:
Merge branch 'develop' into 'fb-PLT-809'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16046181323

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio-enterprise/actions/runs/16048505207
